src/data/checkfiles/default_checkfiles/validate-resc-name-length.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Link } from '@reach/router'
+
+export const validateRescNameLength = {
+    "name": "Each resource has a valid name length",
+    "description": `Checks if the length of each resource is within 63 characters.`,
+    "min_server_version": "4.2.0",
+    "max_server_version": "4.2.10",
```

Comment:
4.2.11 is now released.

Reply:
I changed both `min_server_version` and `max_server_version` to be empty string in all check files for now so zmt won't check it unless it's specified.

Reply:
both are now optional for a checkfile - yes.

but we want to define the ones we ship with minimums of 4.2.0 for now please.